$MirOS: ports/mailnews/pine/patches/patch-imap_src_tmail_Makefile,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/src/tmail/Makefile.orig	Tue Nov 19 00:44:52 2002
+++ imap/src/tmail/Makefile	Wed May  4 23:28:08 2005
@@ -18,7 +18,7 @@
 
 
 C = ../c-client
-CCLIENTLIB = $C/c-client.a
+CCLIENTLIB = -L$C -lc-client
 SHELL = /bin/sh
 
 # Get local definitions from c-client directory
